src/tt.cpp

```diff
@@ -131,11 +131,11 @@ TTEntry* TranspositionTable::probe(const Key key, bool& found) const {
   TTEntry* replace = tte;
   for (int i = 1; i < ClusterSize; ++i)
       // Due to our packed storage format for generation and its cyclic
-      // nature we add 259 (256 is the modulus plus 3 to keep the lowest
+      // nature we add 263 (263 is the modulus plus 7 to keep the lowest
       // two bound bits from affecting the result) to calculate the entry
       // age correctly even after generation8 overflows into the next cycle.
-      if (  replace->depth8 - ((259 + generation8 - replace->genBound8) & 0xFC) * 2
-          >   tte[i].depth8 - ((259 + generation8 -   tte[i].genBound8) & 0xFC) * 2)
```

Comment:
Could you explain why the factor of 2 has been dropped in this patch ?

Reply:
It is miguel-l who do it. I had the same interrogation but as it works, I suppose that it should be OK.

Reply:
From the comment starting line 110, we compute _replace value = depth - 8 * relative age_. The genBound8 previously stored using 6 upper bits so after the mask, we get _4 * relative age_, which we multiply by 2 to get the 8. Now we store using the upper 5 bits instead of 6. Dropping the 2 is to compensate the equivalent shift (i.e. the mask gives us the _8 * relative age_ already).